docs/iris/src/whatsnew/2.3.rst

```diff
@@ -0,0 +1,196 @@
+  coordinate variable will be given a 'climatology' attribute, and the
+  contents of the
+  coordinate's ``bounds`` property are written to a NetCDF boundary variable
+  called 'climatology_bounds'.  These are in place of the more usual NetCDF
+  'bounds' attribute and accompanying 'bnds' boundary variable. See below
+  for an `example from CF conventions <http://cfconventions
+  .org/Data/cf-conventions/cf-conventions-1.7/cf-conventions
+  .html#climatological-statistics>`_:
+
+    .. code-block:: none
+
+        dimensions:
+          time=4;
+          nv=2;
+        variables:
+          float temperature(time,lat,lon);
+            temperature:long_name="surface air temperature";
+            temperature:cell_methods="time: minimum within years time: mean over years";
+            temperature:units="K";
+          double time(time);
+            time:climatology="climatology_bounds";
+            time:units="days since 1960-1-1";
+          double climatology_bounds(time,nv);
+        data:  // time coordinates translated to date/time format
+          time="1960-4-16", "1960-7-16", "1960-10-16", "1961-1-16" ;
+          climatology_bounds="1960-3-1",  "1990-6-1",
+                             "1960-6-1",  "1990-9-1",
+                             "1960-9-1",  "1990-12-1",
+                             "1960-12-1", "1991-3-1" ;
+
+  If a climatological time axis is detected when loading NetCDF -
+  indicated by the format described above - the ``climatological`` property
+  of the Iris coordinate will be set to ``True``.
+
+.. admonition:: New Chunking Strategy
+
+  Iris now makes better choices of Dask chunk sizes when loading from NetCDF
+  files: If a file variable has specified, small chunks, Iris will now choose
+  Dask chunks which are a multiple of these up to a default target size.
+
+  This is particularly relevant to files with an unlimited dimension, which
+  previously could produce a large number of small chunks. This had an adverse
+  effect on performance.
+
+  In addition, Iris now takes its default chunksize from the default configured
+  in Dask itself, i.e. ``dask.config.get('array.chunk-size')``.
+
+.. admonition:: Lazy Statistics
+
+  Several statistical operations can now be done lazily, taking advantage of the
+  performance improvements offered by Dask:
+
+  * :meth:`~iris.cube.Cube.aggregated_by`
+  * :class:`~iris.analysis.RMS` (more detail below)
+  * :class:`~iris.analysis.MEAN`
+
+----
+
+* Cube data equality testing (and hence cube equality) now uses a more relaxed
+  tolerance : This means that some cubes may now test 'equal' that previously
+  did not.
+  Previously, Iris compared cube data arrays using "abs(a - b) < 1.e-8".
+  We now apply the default operation of :func:`numpy.allclose` instead,
+  which is equivalent to "abs(a - b) < (1.e-8 + 1.e-5 * b)".
+* Added support to render HTML for :class:`iris.cube.CubeList`s in Jupyter Notebooks and JupyterLab.
```

Comment:
This may be due to my old version of Sphinx, but it is complaining about this line and in the built copy it doesn't correctly create the link to iris.cube.CubeList

You could also argue that we should use the `~` i.e. :class:`~iris.cube.CubeList`so that it reads more nicely:
`Added support to render HTML for CubeList in Jupyter Notebooks and JupyterLab`
where cube list would link off to the ref for CubeList

Reply:
I caught this one.  I'm pretty sure the cause is the "s" immediately following.
I know there is a way to fix this..

In the 2.2 whatsnew I find : `:class:\`iris.coords.AuxCoord\`'s can now test as 'equal'`
That at least seems to work.

Reply:
Went for @lbdreyer's idea